### PR TITLE
Changing kitten theme

### PR DESCRIPTION
**Pull request by eduardomoroni**
Hey folks sorry for another HUGE PR, but to simplify our hackathon/workshop I thought that I'd be a good idea having a better example on how could we style our app. So I did the ground work for start using [KittenTricks](https://github.com/akveo/kittenTricks). For now on, our folks just need to take a look at KittenTricks repo and copy the desired screen.

This is how is looking like:
image
image

I've also added a little tech debt, I extracted duplicated code into a new component we have to move all tests to this component. I was kinda tired :P

### CODE REVIEW

File: __tests__/modules/authentication/components/LoginForm.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+describe('<LoginForm />', () => {
+  const wrapper = shallow(<LoginForm {...props} />)
+
+  it('Should have a snapshot', () => {
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  it.skip('Should move all tests case from SignInScreen.test.js and SignUpScreen.test.js to HERE', () => {
```
Comment (eduardomoroni) [inline __tests__/modules/authentication/components/LoginForm.test.js]: **TECH DEBT**

File: __tests__/modules/authentication/containers/SignInScreen.test.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ describe('<SignInScreen />', () => {
     expect(wrapper).toMatchSnapshot()
   })
 
-  it('Should update SignInScreen state as text input changes', () => {
+  it.skip('Should update SignInScreen state as text input changes', () => {
```
Comment (eduardomoroni) [inline __tests__/modules/authentication/containers/SignInScreen.test.js]: **TECH DEBT**

File: __tests__/modules/authentication/containers/SignUpScreen.test.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ describe('<SignUpScreen />', () => {
     expect(wrapper).toMatchSnapshot()
   })
 
-  it('Should update SignUpScreen state as text input changes', () => {
+  it.skip('Should update SignUpScreen state as text input changes', () => {
```
Comment (eduardomoroni) [inline __tests__/modules/authentication/containers/SignUpScreen.test.js]: **TECH DEBT**

**Comment by eduardomoroni** [comment]
iOS build is breaking, I'll take a look at this.

**Comment by eduardomoroni** [comment]
Build fixed. `react-native-linear-gradient` is not working on android. Could anyone test it on android? If needed, you can try reinstall it on android following https://github.com/react-native-community/react-native-linear-gradient

**Comment by eduardomoroni** [comment]
Lots of changes folks, and one tech debt 🤒.
Please let me know any concerns.

@taniadgv @gustavomazevedo @viniciusrdacosta

**Comment by viniciusrdacosta** [comment]
> Lots of changes folks, and one tech debt 🤒.
> Please let me know any concerns.

Code climate is complaining about some violations. I believe that some of them are related to ES6 syntax.

Maybe we can fix configuring our `.codeclimate.yml` like this:

```
eslint:
  enabled: true
  config:
    extensions:
    - .es6
    - .js
    - .jsx

ratings:
   paths:
   - "**.es6"
   - "**.js"
   - "**.jsx"
```

The other problems seems to be related to `body of a for-in should be wrapped in an if statement`
The fix should look like this:
```
for (let key in RkTheme.current.fonts.sizes) {
    if(RkTheme.current.fonts.sizes.hasOwnProperty(key)) {
       RkTheme.setType('RkText', key, {
          fontSize: theme => theme.fonts.sizes[key]
       })
    }  
}
```

**Comment by eduardomoroni** [comment]
Hey @viniciusrdacosta, fixed some issues. 
Our real problem is: we're using a design kit which forces us to follow their conventions which are against codeclimate. For now, let keep as it is until we find a good way to fix it.

**Pull request merged**